trap/src/main/java/com/cursorinsight/trap/TrapManager.kt

```diff
@@ -358,7 +368,7 @@ class TrapManager internal constructor(
         }
     }
 
-    override fun onActivityCreated(activity: Activity, savedInstanceState: Bundle?) {
+     override fun onActivityCreated(activity: Activity, savedInstanceState: Bundle?) {
```

Comment:
Unnecessary space here

Reply:
I've force pushed a new version with the fix.